sys/sysmon/sysmon.cpp

```diff
@@ -186,7 +186,7 @@ static void *_thread_status_updater(void *data)
 {
     (void)data;
 
-    ztimer_now_t loop_start_time = ztimer_now(ZTIMER_USEC);
+    ztimer_now_t loop_start_time = ztimer_now(ZTIMER_SEC);
```

Comment:
I thought we should not use `ZTIMER_SEC`.
Is it because the thread period is long enough here?

Reply:
No, just because it is not a critical task needing high accuracy so it is acceptable here to use ZTIMER_SEC